jupyter-book-read-the-docs.rst

```diff
@@ -0,0 +1,94 @@
+.. post:: Nov 11, 2021
+   :tags: feature, jupyter
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Jupyter Book is now supported on Read the Docs,
+      in this post we explain what is needed to make it work.
+
+Jupyter Book ❤️ Read the Docs
```

Comment:
Seems we'd swap this around for our post?

Reply:
I see the marketing reasoning behind it, although I'll try to hint in the text that JB did all the work :)